docs/parameters.rst

```diff
@@ -2,3 +2,276 @@
+*********
+
+All climate models included in OpenSCM must be able to understand and take the following emissions variables as input. Note that adding up all sub-categories of ``Emissions`` will fail as the units will all be different.
+
+- ``Emissions|CO2`` (mass carbon / time)
+- ``Emissions|CH4`` (mass methane / time)
+- ``Emissions|N2O`` (mass nitrous_oxide / time)
+- ``Emissions|SOx`` (mass sulfur_oxide / time)
+- ``Emissions|CO`` (mass carbon_monoxide / time)
+- ``Emissions|NMVOC`` (mass volatile_organic_compound / time)
+- ``Emissions|NOx`` (mass nitrogen_oxide / time)
+- ``Emissions|BC`` (mass black_carbon / time)
+- ``Emissions|OC`` (mass organic_carbon / time)
+- ``Emissions|NH3`` (mass nitrogen / time)
+- ``Emissions|NF3`` (mass NF3 / time)
+- ``Emissions|CF4`` (mass CF4 / time)
+- ``Emissions|C2F6`` (mass C2F6 / time)
+- ``Emissions|C3F8`` (mass C3F8 / time)
+- ``Emissions|cC4F8`` (mass cC4F8 / time)
+- ``Emissions|C4F10`` (mass C4F10 / time)
+- ``Emissions|C5F12`` (mass C5F12 / time)
+- ``Emissions|C6F14`` (mass C6F14 / time)
+- ``Emissions|C7F16`` (mass C7F16 / time)
+- ``Emissions|C8F18`` (mass C8F18 / time)
+- ``Emissions|CCl4`` (mass CCl4 / time)
+- ``Emissions|CHCl3`` (mass CHCl3 / time)
+- ``Emissions|CH2Cl2`` (mass CH2Cl2 / time)
+- ``Emissions|CH3CCl3`` (mass CH3CCl3 / time)
+- ``Emissions|CH3Cl`` (mass CH3Cl / time)
+- ``Emissions|CH3Br`` (mass CH3Br / time)
+- ``Emissions|HFC23`` (mass HFC23 / time)
+- ``Emissions|HFC32`` (mass HFC32 / time)
+- ``Emissions|HFC4310`` (mass HFC4310 / time)
+- ``Emissions|HFC125`` (mass HFC125 / time)
+- ``Emissions|HFC134a`` (mass HFC134a / time)
+- ``Emissions|HFC143a`` (mass HFC143a / time)
+- ``Emissions|HFC152a`` (mass HFC152a / time)
+- ``Emissions|HFC227ea`` (mass HFC227ea / time)
+- ``Emissions|HFC236fa`` (mass HFC236fa / time)
+- ``Emissions|HFC245fa`` (mass HFC245fa / time)
+- ``Emissions|HFC365mfc`` (mass HFC365mfc / time)
+- ``Emissions|CFC11`` (mass CFC11 / time)
+- ``Emissions|CFC12`` (mass CFC12 / time)
+- ``Emissions|CFC113`` (mass CFC113 / time)
+- ``Emissions|CFC114`` (mass CFC114 / time)
+- ``Emissions|CFC115`` (mass CFC115 / time)
+- ``Emissions|HCFC22`` (mass HCFC22 / time)
+- ``Emissions|HCFC141b`` (mass HCFC141b / time)
+- ``Emissions|HCFC142b`` (mass HCFC142b / time)
+- ``Emissions|SF6`` (mass SF6 / time)
+- ``Emissions|SO2F2`` (mass SO2F2 / time)
+- ``Emissions|Halon1202`` (mass Halon1202 / time)
+- ``Emissions|Halon1211`` (mass Halon1211 / time)
+- ``Emissions|Halon1301`` (mass Halon1301 / time)
+- ``Emissions|Halon2402`` (mass Halon2402 / time)
+
+
+Concentrations
+**************
+
+Atmospheric concentrations should be of the form ``Atmospheric Concentrations|XXX`` e.g. ``Atmospheric Concentrations|CO2``.
+OpenSCM models should understand all of the following variables.
+Note that adding up all sub-categories of ``Atmospheric Concentrations`` will either fail as the units will not all be the same or will produce a basically nonsense number (but still be possible).
```

Comment:
Do we need this comment? Maybe generalize it and move to the general "Aggregation" section (see my PR).

Reply:
think this will be fixed in #150 no?